tools/coverage/design.md

```diff
@@ -0,0 +1,36 @@
+# Design of Test Coverage Tool
+
+![design.svg](design.svg)
+
+We pack the test coverage tool in a container, that is triggered by prow. It runs test coverage profiling on target repository. Afterward, it calculates coverage change or summarizes coverage data, depends on workflow type, explained below.  
+
+## Post-submit workflow
+
+Produces and stores coverage profile, as master branch, for later presubmit jobs to compare against
+
+1. A PR is merged.
+1. Post-submit prow job started.
+1. Test coverage profile generated. Completion marker generated upon successful run.
```

Comment:
This is the only place the "completion marker" is mentioned. Please explain what it is and why it's important.